packages/oca/src/types/TypeEnums.ts

```diff
@@ -0,0 +1,20 @@
+export enum BaseType {
+  Binary = 'Binary',
+  Text = 'Text',
+  DateTime = 'DateTime',
+  Numeric = 'Numeric',
+  DateInt = 'DateInt',
+}
+
+export enum BaseOverlayType {
+  Base10 = 'spec/capture_base/1.0',
```

Comment:
Small nitpick. I’d call this `CaptureBase10` since we’re renaming.

Reply:
Good point. Changing it now.